styles/styles.css

```diff
@@ -406,6 +406,14 @@ main img {
 
 /* START Section Styles START */
 
+main .section .section-container h6 {
+  margin-bottom: 0.5rem;
+}
+
+main .section .section-container p {
```

Comment:
Is this universally applicable in the original site i.e. majority paragraphs needing this much padding?

Reply:
yes, checked across few pages and observed this common pattern hence added it as global rule.

Reply:
Could you check with [global styles of the live site](https://www.sunstar.com/wp-content/themes/sunstar-group/public/css/app.css)? I see the following global style in it 
```
p {
    margin-top: 0;
    margin-bottom: 1rem
}
```